spec/feature/user_can_log_in_feature_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+feature 'User can log in' do
+    context 'Sign in to account'
+    before do
+      create(:user, email: 'user@mail.com', password: '12345678')  
+        visit user_session_path
+end
+
+    it 'displays a log in button' do
+        expect(page).to have_content 'Login'
+        click_on 'Login'
+    end
+
+    it 'displays a email form' do
+        expect(page).to have_content 'Email'    
+    end
+
+    it 'fills in email and password' do
+        fill_in "Email", :with => 'user@mail.com'
+        fill_in "Password", :with => '12345678'
+        click_on "Log in"
+            expect(page).to have_content 'Signed in successfully.'
+    end
+
+    it 'If no email or password is entered' do
+    fill_in 'Email', :with => ''
+    fill_in 'Password', :with => ''
+    click_on 'Log in'
+        expect(page).to have_content 'Invalid Email or password.'
+    end
+
+
```

Comment:
What happens if a user tries to log in with the wrong password? Or the wrong email? You should write a test for that and have a more realistic scenario than empty email and empty password.